File: core/src/org/rexellentgames/dungeon/debug/LightCommand.java
package org.rexellentgames.dungeon.debug;

import org.rexellentgames.dungeon.entity.level.LightLevel;

public class LightCommand extends ConsoleCommand {
	{
		name = "/light";
		shortName = "/lt";
		description = "toggles light";
	}

	@Override
	public void run(Console console, String[] args) {
		LightLevel.LIGHT = !LightLevel.LIGHT;
	}
}

File: core/src/org/rexellentgames/dungeon/debug/LightCommand.kt
package org.rexellentgames.dungeon.debug

import org.rexellentgames.dungeon.entity.level.LightLevel

class LightCommand : ConsoleCommand("/light", "/lt", "Toggles light") {
    override fun run(console: Console, args: Array<String>) {
        LightLevel.LIGHT = !LightLevel.LIGHT
    }
}